### DISCUSSION TITLE
Include the 11m band

### DISCUSSION THREAD

**Discussion by dgdavid** [magicbug/Cloudlog#853] [Ideas / Feature Requests]
Hi there!

@magicbug, maybe not a feature request itself, but I'm wondering if there is any chance to include the [Citizens Band (11m)](https://en.wikipedia.org/wiki/Citizens_band_radio) in the bands selector.

As far as I can see, it is not in the [ADIF 3.1.1 specification](https://www.adif.org/311/ADIF_311.htm) (see _III.B.4 Band Enumeration_) and it could have implications when exporting to external services like eQSL, although this can be addressed by only exporting ADIF compliant QSOs (I guess)

Since bands do not come from the database yet, maybe the feature request should be to move them to its table and be able to add/remove/enable/disable the desired ones, similar to the behavior of the mode.

But  just adding it to the selector as a first step would be enough. At least here, in Spain, such band still being quite used. In fact, my father's equipment right now are a 2m and 11m radios (if I'm not wrong).

What do you think? Would be possible?

As a reminder, please take into account that I'm not a ham radio and I'm getting involved in the project just to set such interesting tool for my father. So, although I try to be documented before raising questions, my apologies in advance if I overlook something or I make no-sense requests.

**Comment by dg9vh** [comment]
11m is not an Amateur Radio Band... so I don't think it makes sense to put it into the list.

If Peter sees it different and says cloudlog is not only for Amateur Radio purpose but also for CB and Freenet and PMR446 - adding them will be an option (but I would prefer having it configurable then).

**Reply by dgdavid** [reply]
> 11m is not an Amateur Radio Band... so I don't think it makes sense to put it into the list.
> 

I didn't know it, sorry. BTW, what is the main difference? (Just if you have time to explain it)

> If Peter sees it different and says cloudlog is not only for Amateur Radio purpose but also for CB and Freenet and PMR446 -

First time I read about the two latest 😅 

> adding them will be an option (but I would prefer having it configurable then).

I'll appreciated. Sure, having it configurable would be better.

**Comment by magicbug** [comment]
Currently not planning on supporting non ham bands as it just adds filtering headache.

**Comment by dgdavid** [comment]
Thanks for your quick replies, guys.

@magicbug, could you please elaborate why do you think filtering is a _headache_?
